R/rrq_controller.R

```diff
@@ -999,11 +999,11 @@ rrq_controller <- R6::R6Class(
     ##'   wait for the background process to respond to SIGTERM before
     ##'   we stop the worker. Only used for tasks queued with
     ##'   `separate_process` `TRUE`.
-    worker_config_save = function(name, time_poll = NULL, timeout = NULL,
+    worker_config_save = function(name, time_poll = NULL, timeout_idle = NULL,
                                   queue = NULL, heartbeat_period = NULL,
                                   verbose = NULL, overwrite = TRUE,
                                   timeout_poll = 1, timeout_die = 2) {
-      worker_config_save(self$con, private$keys, name, time_poll, timeout,
+    worker_config_save(self$con, private$keys, name, time_poll, timeout_idle,
```

Comment:
```suggestion
      worker_config_save(self$con, private$keys, name, time_poll, timeout_idle,
```

Something happened to the indenting here...